tests/test_tally_slice_merge/results_true.dat

```diff
@@ -1,36 +1,37 @@
-0          0.00e+00           6.25e-07    27   U-235  nu-fission 1.08e-01   1.76e-02   energy low [MeV]  energy high [MeV]  cell nuclide    score     mean  std. dev.
-0          0.00e+00           6.25e-07    27   U-238  fission 6.14e-08   9.64e-09   energy low [MeV]  energy high [MeV]  cell nuclide       score     mean  std. dev.
-0          0.00e+00           6.25e-07    27   U-238  nu-fission 1.53e-07   2.40e-08   energy low [MeV]  energy high [MeV]  cell nuclide    score     mean  std. dev.
-0          6.25e-07           2.00e+01    27   U-235  fission 1.39e-02   1.06e-03   energy low [MeV]  energy high [MeV]  cell nuclide       score     mean  std. dev.
-0          6.25e-07           2.00e+01    27   U-235  nu-fission 3.40e-02   2.61e-03   energy low [MeV]  energy high [MeV]  cell nuclide    score     mean  std. dev.
-0          6.25e-07           2.00e+01    27   U-238  fission 9.72e-03   1.21e-03   energy low [MeV]  energy high [MeV]  cell nuclide       score     mean  std. dev.
-0          6.25e-07           2.00e+01    27   U-238  nu-fission 2.71e-02   3.80e-03    energy low [MeV]  energy high [MeV]  cell nuclide       score     mean  std. dev.
-0           0.00e+00           6.25e-07    21   U-235     fission 1.08e-01   7.94e-03
-1           0.00e+00           6.25e-07    21   U-235  nu-fission 2.64e-01   1.94e-02
-2           0.00e+00           6.25e-07    21   U-238     fission 1.51e-07   1.00e-08
-3           0.00e+00           6.25e-07    21   U-238  nu-fission 3.76e-07   2.50e-08
-4           0.00e+00           6.25e-07    27   U-235     fission 4.43e-02   7.21e-03
-5           0.00e+00           6.25e-07    27   U-235  nu-fission 1.08e-01   1.76e-02
-6           0.00e+00           6.25e-07    27   U-238     fission 6.14e-08   9.64e-09
-7           0.00e+00           6.25e-07    27   U-238  nu-fission 1.53e-07   2.40e-08
-8           6.25e-07           2.00e+01    21   U-235     fission 3.12e-02   2.56e-03
-9           6.25e-07           2.00e+01    21   U-235  nu-fission 7.65e-02   6.24e-03
-10          6.25e-07           2.00e+01    21   U-238     fission 2.00e-02   1.30e-03
-11          6.25e-07           2.00e+01    21   U-238  nu-fission 5.56e-02   3.78e-03
-12          6.25e-07           2.00e+01    27   U-235     fission 1.39e-02   1.06e-03
-13          6.25e-07           2.00e+01    27   U-235  nu-fission 3.40e-02   2.61e-03
-14          6.25e-07           2.00e+01    27   U-238     fission 9.72e-03   1.21e-03
-15          6.25e-07           2.00e+01    27   U-238  nu-fission 2.71e-02   3.80e-03         sum(distribcell)  energy low [MeV]  energy high [MeV] nuclide       score     mean  std. dev.
+   cell  energy low [MeV]  energy high [MeV] nuclide    score     mean  std. dev.
+0    21          0.00e+00           6.25e-07   U-235  fission 1.08e-01   7.94e-03   cell  energy low [MeV]  energy high [MeV] nuclide       score     mean  std. dev.
```

Comment:
The old results have the cell ID, but it was after the energy columns. Fixing the typo on lines 41-42 of `test_tally_slice_merge.py` results in the energy and cell filters being added in the opposite order so now the cell ID column shows up before the energy columns.